task_12/src/css/desktop/main.css

```diff
@@ -0,0 +1,64 @@
+main {
+  padding: 40px 30px 0;
+  font-family: "Ubuntu", sans-serif; }
+
+.button {
+  position: relative;
+  display: block;
+  overflow: visible;
+  /* IE9 & 10 */
+  max-width: 200px;
+  cursor: pointer;
+  transition: 500ms linear;
+  color: #fef1e0;
+  border: 1px solid #fef1e0;
+  border-radius: 5px;
+  outline: none;
+  background-color: #c29c5b;
+  box-shadow: inset 1px 1px 0 rgba(255, 255, 255, 0.25), inset 0 0 6px #a23227, inset 0 80px 80px -40px #ac3223, 1px 1px 3px rgba(0, 0, 0, 0.75);
+  /* shadow */
+  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.75);
+  font-size: 14px; }
+  .button:hover {
```

Comment:
and here

Reply:
<img width="631" alt="screen shot 2018-02-05 at 15 38 11" src="https://user-images.githubusercontent.com/24464487/35807160-9b81f866-0a8a-11e8-80c1-e4b2ecad1df6.png">
The same situation